### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Tobinchilongo**
## What does this PR do?

- This PR is raised to create a movie listing app with comment and likes options

## Description of the task to be completed

- Create the home page
- Set up the project with Webpack and Jest
- Display comments for a given item on the Comments pop-up
- Display Comments pop up with selected item's details
- Add comments counter
- Add footer
- Add header
- Display the number of likes for each item on the Homepage
- Display list of items on the Homepage
- Create feature: add a new comment
- Create feature: add new like
- Add all items counter on the Homepage
- Add tests for items counter
- Add tests for comments counter

## How should it be tested

- clone the repository by running
``` git clone https://github.com/Tobinchilongo/JavaScript_Capstone.git ```
- navigate to the folder
``` cd JavaScript_Capstone ```
- Install packages
``` npm install ```
- Run the app
``` npm start ```
- Run the test 
`npm run test`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,64 @@
...
+- Webpack
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://thecapstone-project-js.netlify.app/)
 
+## Video Dem
+[Video Demo Link](https://www.loom.com/share/1491c0c0ae6344cd9c4b67109e48676d)
```
Comment (Dipeshtwis) [inline README.md:21]: - [ ]  All the members in the video should appear in the video and talk about the project.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Tobinchilongo & @selma-belhadj, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: All the tests are passing.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (selma-belhadj) [reply README.md]: Hello @Dipeshtwis, thank you for your review and we have implemented the required changes.

**Review by Lordkaito — APPROVED** [review]
Hi @Tobinchilongo and @selma-belhadj, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

- [ ] [OPTIONAL] As an optional feature, I would recommend to add a simple `overflow: hidden` to you app whenever you click on the comments button. This is because when you click the comments button for any movie, the user can scroll the comments or can still scroll the main page, which can lead to a bad user experience. 
Whether you decide to implement this feature or not, this was an awesome project, good job! 🏆 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**